src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UseCountProperly.cs

```diff
@@ -442,16 +443,15 @@ private static void ReportCA1836(OperationAnalysisContext context, string operat
                     else
                     {
                         if (allowedTypesForCA1836.Contains(type.OriginalDefinition) &&
-                            TypeContainsVisibleProperty(context, type, IsEmpty, SpecialType.System_Boolean, out ISymbol? isEmptyPropertySymbol) &&
-                            !IsPropertyGetOfIsEmptyUsingThisInstance(context, invocationOperation, isEmptyPropertySymbol!))
```

Comment:
The check was removed from here as it's now contained in the renamed method `TypeContainsVisiblePropertyWithoutCircularity`. I prefer to move the check rather than repeatedly adding it in all call-sites.